sklearn/ensemble/bagging.py

```diff
@@ -52,6 +53,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
     bootstrap_features = ensemble.bootstrap_features
     support_sample_weight = has_fit_parameter(ensemble.base_estimator_,
                                               "sample_weight")
+    # Logistic regression does not support sample weights with liblinear
+    if (isinstance(ensemble.base_estimator_,LogisticRegression) and 
```

Comment:
Hm.... I see about four options: 1) do an if for liblinear here [as is now] 2) make it a ``try`` / ``except`` instead of trying to figure out if sample weights will be supported or not 3) add some interface that will allow you to inspect if a model supports sample weights or not [please not] 4) add it for all solvers.

It think 1) and 2) are the most practical ones. I somewhat favor 2 over 1.